cedar-lean/Cedar/Data/LT.lean

```diff
@@ -145,36 +145,36 @@ theorem List.lt_conn [LT α] [StrictLT α] {xs ys : List α} :
   intro h₁
   by_contra h₂
   simp [not_or] at h₂
-  rcases h₂ with ⟨h₂, h₃⟩
+  have ⟨h₂, h₃⟩ := h₂
   cases xs <;> cases ys
-  case intro.nil.nil => contradiction
-  case intro.nil.cons _ _ xhd xtl =>
-    rcases (List.lt.nil xhd xtl) with h₄
+  case nil.nil => contradiction
+  case nil.cons _ _ xhd xtl _ =>
+    have h₄ := List.lt.nil xhd xtl
```

Comment:
What's the rationale for the case name changes here and in the rest of the PR? (Just want to check if there's an established style rule to follow.)

Reply:
We don't choose the case names.  The adjustments to case names are all in response to errors about case-name-not-found.  As you can see, `rcases` was polluting our case names with uninformative garbage like `intro.`, which we no longer `have` (lol) to put up with.  I also touched in this in the PR description